index.html

```diff
@@ -5,74 +5,118 @@
+
+          <p>Using an ice cream machine, pour liquids into the frozen basin and process according to manufacturer instructions, i.e., let spin and thicken for 20 minutes befre adding mascarpone, fig jam mixture, and the nuts. Continue to process for +/- 10 minutes. Pour semi-frozen mixture into a pyrex dish or glass tupperware. Free for at least two hours before serving.</p>
+        </div>
+     
+        <section class="grid-100 after-mobile-100">
+          <div class="recipe">
+            <form class="ingredients">
+              <input type="checkbox" name="chk" value="" unchecked> 1 1/2 cups milk</br>
+              <input type="checkbox" name="chk" value="" unchecked> 1/2 cup mascarpone</br>
+              <input type="checkbox" name="chk" value="" unchecked> 1/2 tsp pink salt</br>
+              <input type="checkbox" name="chk" value="" unchecked> 1 lb Black Mission Figs</br>
+              <input type="checkbox" name="chk" value="" unchecked> 1/2 cup brown sugar</br>
+              <input type="checkbox" name="chk" value="" unchecked> 2-4 tbsp water</br>
+              <input type="checkbox" name="chk" value="" unchecked> 1 1/2 cups heavy cream</br>
+              <input type="checkbox" name="chk" value="" checked> 1/3 granulated sugar</br>
+              <input type="checkbox" name="chk" value="" checked> 2 egg yolks</br>
+              <input type="checkbox" name="chk" value="" unchecked> 1 lemon, juiced</br>
+              <input type="checkbox" name="chk" value="" unchecked> 2 tbsp butter</br>
+              <input type="checkbox" name="chk" value="" unchecked> 1 cup honey roasted pecans, roughly chopped</br>
+            </form>
+          </div>
+        </section>
+      </article>
+    </main>
+    <div class="grid-100">
+      <img class="horizonbreak" src="images/horizonline.png"/>
+    </div>
+    <section class="grid-100">
+      <div class="grid-60 mobile-grid-100">
+        <img class="avatar" src="images/headshot1.png" alt="Vanessa Stevenson"/>
+        <h1 class="author-name">Vanessa Stevenson</h1>
+        <h2 class="author-bio">Food enthusiast, photography fan. Add a pinch of raw foodism and that's pretty much who I am.</h2>
+      </div>
+      <div class="grid-40 mobile-grid-100">
+        <button class="share-recipe">Share Recipe</button>
+      </div>
+    </section>
   </body>
   <footer>
-    <img src="images/headshot1.png" alt="Vanessa Stevenson"/>
-    <h1>Vanessa Stevenson</h1>
-    <p>Food enthusiast, photography fan. Add a ping of raw foodism and that's pretty much who I am.</p>
-    <img src="images/btmlogo.png"/>
-    <p>Delicious &copy 2013 - All Rights Reserved.<br>Proudly published with Ghost.</p>
+    <div class="grid-100 mobile-grid-100">
+      <img src="images/btmlogo.png"/>
+      <p>Delicious &copy 2013 - All Rights Reserved.<br>Proudly published with Ghost.</p>
```

Comment:
Like I mentioned in class today, use css to achieve what you're trying to achieve with the br tag.